cascade.py

```
@@ -1,146 +1,246 @@
+import config
+import sys, gzip, shutil, os, cv2, time, csv, telegram, requests, argparse, asyncio
+import logging
+from logging.handlers import RotatingFileHandler
 import numpy as np
 from pathlib import Path
-import os, cv2, time, csv, sys, gc
-import pytz
 from datetime import datetime
 from collections import deque
-from threading import Thread
+from threading import Event, Lock, Thread
```

Ruff (0.11.9)

`threading.Event` imported but unused

Remove unused import

(F401)